CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Changed
 - Updated class calls to accommodate the latest release 0.13.1 of SwissPaymentSlipPdf
-
+- Implemented the umlaut fix
+- Changed the output code block to facilitate sprintf syntax
+- Added created example pdf files to .gitignore
+- Updated CHANGELOG
 ### Fixed
 - Changed convertColor2Rgb function to protected instead of private
+- Fixed mix ups with TCPDF code base were i was able to find them
```

Comment:
The CHANGELOG is not about you and what you did but for other people to see in a brief way to see what has changed in  a specific version. Please also remove this line. I'll update the change log.